app/javascript/modules/listingDetails/ListingDetailsUnitAccordions.tsx

```diff
@@ -47,8 +47,9 @@ const getTableData = (units: RailsListingUnits[]) =>
     area: {
       content: (
         <>
-          <span className="font-semibold">{unit.Unit_Square_Footage}</span>
-          {t("listings.features.sqft")}
+          <span className="font-semibold">{unit.Unit_Square_Footage}</span>{" "}
+          <span aria-hidden="true">{t("listings.features.sqft")}</span>
+          <span className="sr-only">{t("listings.features.squareFeet")}</span>
```

Comment:
reads out as expected on my mac w/ browser tools in chrome and safari. i also tested with [browser stack](https://www.browserstack.com/guide/test-websites-with-screen-readers) and this did not read out 'square feet', but it's possible i'm doing it wrong. i'm also trying to get access to the assistiv labs tooling. will let you know how that goes

Reply:
Looks like NVDA is ok, but Jaws is skipping over the square feet

https://user-images.githubusercontent.com/98352600/215902490-01898f15-ca09-4daa-9e10-c90f0a49ad84.mov

https://user-images.githubusercontent.com/98352600/215902509-95a8b2c8-dbed-4f4c-8150-7c7a520e0dd9.mov